_posts/2021-01-xx-release-1.12.1.md

```diff
@@ -0,0 +1,300 @@
+---
+layout: post
+title:  "Apache Flink 1.12.1 Released"
+date:   2021-01-xx 00:00:00
+categories: news
+authors:
+- xtsong:
+  name: "Xintong Song"
+  twitter: "xtsong"
+---
+
+The Apache Flink community released the first bugfix version of the Apache Flink 1.12 series.
+
+This release includes 81 fixes and minor improvements for Flink 1.12.0. The list below includes a detailed list of all fixes and improvements.
```

Comment:
I guess the number 81 is outdated? I can see 107 tickets in https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12349459.

Besides that, there are 109 tickets attached in this PR.

Reply:
Thanks for pointing out. I think the 107/109 tickets include unresolved issues. I'll postpone them to 1.12.2 and update the PR once the RC is approved.